app/models/payment/go_cardless/transaction.rb

```diff
@@ -81,8 +81,10 @@ def publish_failed_subscription_charge
     ChampaignQueue.push({
       type: 'subscription-payment',
       params: {
+        created_at: created_at,
         recurring_id: subscription.go_cardless_id,
-        success: 0
+        success: 0,
+        status: 'failed'
```

Comment:
Yep, that makes sense. But this is how it's been implemented in Champaign at the moment and I don't have the time to refactor it right now to get the legacy data through today.